client/src/assets/icons/help.tsx

```diff
@@ -3,18 +3,45 @@ import React from 'react';
 function Help(
   props: JSX.IntrinsicAttributes & React.SVGProps<SVGSVGElement>
 ): JSX.Element {
+  const wrapper = {
+    display: 'grid',
+    placeItems: 'center',
+    position: 'relative',
+    padding: 'inherit'
+  } as React.CSSProperties;
+
+  const speechBubbleIcon = {
+    position: 'absolute'
+  } as React.CSSProperties;
+
+  const questionMarkIcon = {
+    position: 'absolute',
+    top: '-3px',
+    width: '9px'
+  } as React.CSSProperties;
```

Comment:
having the values inside the component, make it reassigned every time the component render, can you move the values out of the help function.

Reply:
I don't think there will be a need for the props, but let's see how they implement the changes first.

It isn't a big deal, I can create a quick PR after the issue is closed, if the props aren't accessed.